04_setup_ironic.sh

```diff
@@ -29,6 +29,20 @@ jq -s '.[0] * .[1]' ${REGISTRY_AUTH_FILE} ${REGISTRY_CREDS} > ${COMBINED_AUTH_FI
 DOCKERFILE=$(mktemp --tmpdir "release-update--XXXXXXXXXX")
 _tmpfiles="$_tmpfiles $DOCKERFILE"
 echo "FROM $OPENSHIFT_RELEASE_IMAGE" > $DOCKERFILE
+
+# To build custom images is highly recommended to build a base image first.
+# Build a base image if we set a custom repo file in the config file and
+# the file exists
+if [[ -n ${CUSTOM_REPO_FILE:-} ]]; then
```

Comment:
Given the kind of check probably the default isn't necessary?

Reply:
if we don't set the default here the check will always fail because of "unbound variable"

Reply:
AFAIK -n should not fail in case the variable is not set.

Reply:
We have `set -u` so it will fail if the variable is not set.